utils.groovy

```diff
@@ -131,6 +131,10 @@ def add_hotfix_parameters_if_supported() {
     return []
 }
 
+def is_rhcos_jenkins() {
+    return env.JENKINS_URL && env.JENKINS_URL.contains('rhcos')
```

Comment:
I did something similar in https://github.com/coreos/fedora-coreos-pipeline/pull/1197 but it for detecting when we were running against FCOS.


```
if (pipecfg.source_config.url != 'https://github.com/coreos/fedora-coreos-config')
    // Only skip builds in FCOS for now                                           
    return false                                                                  
}                                                                                 

```


@jlebon @marmijo @aaradhak is there a method you prefer here?

Reply:
Hmm, can we shorten it to `allow_missing_arches_default`?

Reply:
> Hmm, can we shorten it to `allow_missing_arches_default`?

yeah, but it's slightly more complicated than that implies. Right now where we are using it is when triggering a later job rather than actually setting the default for that parameter in that job.

so I do like the shorter name, but it doesn't tell the whole story.

Reply:
Yeah, naming is tricky. I'm OK with something wordier, but at some point it's better to capture the nuance in a docstring rather than trying to capture it all in the variable name.

Reply:
Okay I will go-ahead and do that thank you!